### DISCUSSION TITLE
Casl/vue with Vue2

### DISCUSSION THREAD

**Discussion by bakhti-ai** [stalniy/casl#478] [Q&A] [answered]
Hi,  I am trying to use @casl/vue.1.x with Vue-2, but not working, can anyone help with this?
Here is the link for detailed explanation of problem 
[https://stackoverflow.com/questions/66527075/casl-vue-can-function-is-returning-false-but-user-has-that-permission](url)
@stalniy

**Comment by stalniy** [comment]
Hi 

did you read about debugging in docs? - https://casl.js.org/v5/en/advanced/debugging-testing

**Reply by bakhti-ai** [reply]
Yes, I have read debugging part and tried `ability.relevantRuleFor` method, it is always returning `null`. But there are my rules inside `ability` object

**Comment by bakhti-ai** [comment, marked as answer]
I was using `@casl/ability@5.2.2` and `@casl/vue@1.2.2`
As said in the comments by @stalniy on SO [https://stackoverflow.com/q/66527075/8770647](url)

> If you get null for `relevantRuleFor` it means that casl doesn’t a rule for provided action/subject pair. From what I can tell is that the shape of `rule` object is wrong. actions was deprecated in v4 and removed in v5. Try to replace `actions` with `action`

After changing `actions` to `action` that worked.

Thanks @stalniy 
Casl is a great library
